cmd/tasks/claim.go

```diff
@@ -74,9 +74,9 @@ func ClaimTaskCmd(c client.Client) *cobra.Command {
 	}
 
 	// Define command flags
-	cmd.Flags().IntVarP(&counter, "counter", "c", 0, "The task counter")
-	cmd.Flags().StringVarP(&processId, "process-id", "p", "", "Unique process ID that identifies the claimer")
-	cmd.Flags().DurationVarP(&ttl, "ttl", "t", 0, "Time to live in milliseconds")
+	cmd.Flags().IntVarP(&counter, "counter", "c", 0, "task counter")
+	cmd.Flags().StringVarP(&processId, "process-id", "p", "", "unique id that identifies the claimer")
+	cmd.Flags().DurationVarP(&ttl, "ttl", "t", 0, "task time to live")
```

Comment:
I think we should keep the "in milliseconds" somewhere.

Reply:
Yes for sure in our API docs! Here in the CLI you can specify a duration, so `1m`, `2h`, `3d` etc - the unit is up to you.